### PR TITLE
fetch data from API and create the placeToVisitOverview component

### PR DESCRIPTION
**Pull request by binhtendo**
would you please be so kind and check my PR

### CODE REVIEW

File: userstories.excalidraw [hunk 1]
```diff
@@ -0,0 +1,2555 @@
+{
```
Comment (B-Homsi) [inline userstories.excalidraw:1]: Is this file intented to be here?

**Review by Kheirah — CHANGES_REQUESTED** [review]
Good start!

Check the comments and adjust your code where necessary.

File: userstories.excalidraw (hunk 1 above)
Comment (binhtendo) [reply userstories.excalidraw:1]: actually not. i created my userstories with the excalidraw extension @VS Code

File: components/placesToVisitOverview/index.js [hunk 2]
```diff
@@ -51,23 +29,31 @@ export default function PlacesToVisitOverview() {
...
-              }}
-            >
-              {place.isFavorite ? "★" : "☆"}
-            </button>
-          </li>
+          <Place
+            key={place.properties.xid}
+            name={place.properties.name}
+            onFavorite={toggleFavorite}
+            xid={place} // hier sollte xid={place.properties.xid} sein
```
Comment (TuncAlt) [inline components/placesToVisitOverview/index.js:55]: should this not be {place.properties.xid}?

**Review by TuncAlt — CHANGES_REQUESTED** [review]

File: components/placesToVisitOverview/index.js (hunk 2 above)
Comment (binhtendo) [reply components/placesToVisitOverview/index.js:55]: you are absolutely right. will change it

**Review by TuncAlt — APPROVED** [review]
Looks good for now, don't forget the error message.

File: utils/api.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      apiKey;
+
+    if (query !== undefined) {
+      otmAPI += "&" + query;
+    }
+
+    fetch(otmAPI)
+      .then((response) => response.json())
+      .then((data) => resolve(data))
+      .catch(function (err) {
```
[the remark this answers is not in the document]
Comment (binhtendo) [reply utils/api.js:18]: done

**Pull request merged**